### PR TITLE
What open source means to me :)

### PR DESCRIPTION
**Pull request by CodeMaxx**
Open Source ftw!

@nickdesaulniers

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -431,3 +431,8 @@ Open source é uma filosofia de vida. O gosto de poder contribuir com algo que a
 ###[Veit Heller](https://veitheller.de)
 
 Für mich ist Open Source der kreativste und freiste Weg, mich als Entwickler auszudrücken. Nur durch freien Quellcode werden auch die Gedanken und das Ergebnis wirklich frei, offen und - letztendlich - glaubwürdig. Freie Software eliminiert alle Grenzen zwischen Menschen, erlaubt es uns gemeinsam zu basteln und zu erschaffen, wie wir es uns wünschen - und hoffentlich so gut wir können. Durch die inherente Motivation des Aus-Sich-Selbst-Heraus-Schaffens wird Software besser, werden kreative Energien freigesetzt und Gemeinsamkeit geschaffen. Ob das Ziel Selbstzweck, Tatendrang, Geschäfts- oder Gemeinwohl ist, jedes Programm profitiert von offenem Quellcode. Es wird Zeit für uns zu erkennen, dass Komplexität nur durch Kooperation handhabbar ist.
+
+###[Akash Trehan](https://codemaxx.github.io/)
+
+For me Open Source represents the power of collaboration. Lots and lots of small contributions bubble up something that's useful. Open source is fun and it works. It gives me control over the sofware I use.<br>
```
Comment (justinclift) [inline README.md]: Typo. :wink:

"sofware" is missing a "t".  In theory, you probably don't need the "<br>" tag either.  Just add a blank line instead. :smile:

File: README.md [hunk 2]
```diff
@@ -431,3 +431,8 @@ Open source é uma filosofia de vida. O gosto de poder contribuir com algo que a
...
+The adrenaline rush when you code finally starts working and your PR is merged, thats priceless. The discussions are really productive and there's a lot that I learn from others. This helps me expand my horizons while doing something I really love, to code. It also gives me a unique satifaction to know my code would help millions if not more. Thats what keeps me ticking!
```
Comment (justinclift) [inline README.md]: Also: "you code" → "your code"

File: README.md (hunk 1 above)
Comment (CodeMaxx) [inline README.md]: Thanks! Fixing it.
The tag in your comment got converted 😛

File: README.md (hunk 2 above)
Comment (CodeMaxx) [inline README.md]: Fixed.

File: README.md (hunk 1 above)
Comment (justinclift) [inline README.md]: Gah.  Yeah, I really should have hit the preview button eh? :smile:

**Comment by justinclift** [comment]
Thanks @CodeMaxx, merged it. :smile:

**Pull request merged**